src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -118,7 +118,7 @@ type [<AllowNullLiteral>] internal DynamicViewModel<'model, 'msg>
           | Error e ->
               match e with
               | GetError.OneWayToSource -> log.LogError("[{BindingNameChain}] TryGetMember FAILED: Binding {BindingName} is read-only", nameChain, binder.Name)
-              | GetError.SubModelSelectedItem d -> log.LogError("[{BindingNameChain}] TryGetMember FAILED: Failed to find an element of the SubModelSeq binding {SubModelSeqBindingName} with ID {ID} in the getter for the binding {SubModelSelectedItemName}", d.NameChain, d.SubModelSeqBindingName, d.Id)
+              | GetError.SubModelSelectedItem d -> log.LogError("[{BindingNameChain}] TryGetMember FAILED: Failed to find an element of the SubModelSeq binding {SubModelSeqBindingName} with ID {ID} in the getter for the binding {BindingName}", d.NameChain, d.SubModelSeqBindingName, d.Id, binder.Name)
```

Comment:
> - ...{SubModelSelectedItemName}...
> + ...{BindingName}...

I had picked `SubModelSelectedItemName` since there are two bindings involved, and I didn't want any confusion.  Although, the other binding is called `SubModelSeqBindingName`, so by process of elimination, using `BindingName` here is still unambiguous.

Do you prefer `BindingName` because that is the name used in every other log event?

Reply:
While the other interpolations are directly named after their fields on `d`, it was difficult, as one unfamiliar with the code, to discover what argument was supposed to go with `SubModelSelectedItemName`. I thought that the name `BindingName` more clearly indicated that it was supposed to take `binder.Name`. But then, code isn't necessarily optimized for those new to the project. What think you?

Reply:
Also, I thought that, since this `match` branch has to do mainly with `subModelSelectedItem` bindings, the type of the matched binding is probably obvious.

Reply:
> Do you prefer BindingName because that is the name used in every other log event?

Yes. If you use a logging system like seq, and you want to search for all log events for a given binding name, you would have to include this one explicitly for this log case in your query expression. It's *much better* that they be named similarly for this purpose.